lighthouse-core/test/audits/__snapshots__/metrics-test.js.snap

```diff
@@ -35,5 +34,6 @@ Object {
   "observedTraceEndTs": 225426711887,
   "speedIndex": 1657,
   "speedIndexTs": undefined,
+  "totalBlockingTime": 726,
```

Comment:
now that the clipping order has moved back to the original, the fact that this changes is unexpected to me.

could we perhaps add a new unit test case that would be different between the two versions to capture this more explicitly? 

from my reading it should only be different in cases that I thought to be impossible

Reply:
Hm very good point - thanks for flagging this. I looked into it, and looks like the impossible is happening - TTI is in the middle of a task for the lantern test case (and it's not at FMP or DCL). Right now I'm trying to understand why that is the case. 

Side note: It looks like lantern TTI does not consider DCL. I see this at line 70 of lantern-interactive.js. Is this expected?
```js
    const lastTaskAt = LanternInteractive.getLastLongTaskEndTime(simulationResult.nodeTimings);
    const minimumTime = extras.optimistic
      ? extras.fmpResult.optimisticEstimate.timeInMs
      : extras.fmpResult.pessimisticEstimate.timeInMs;
    return {
      timeInMs: Math.max(minimumTime, lastTaskAt),
      nodeTimings: simulationResult.nodeTimings,
    };
```

Reply:
Oh, right because we are using the TTI estimate `timeInMs` of a potentially different graph so it is possible.

Re: DCL, Yeah there's not a reliable way we have to predict when DCL will occur under a hypothetical connection type so it's not considered.

AI from my perspective: we add a test case *just* exercising the TTI clipping behavior with a comment about why this can happen

Reply:
Ah right. That sgtm: added test case just for TTI clipping.